pkg/util/runner.go

```diff
@@ -33,6 +33,7 @@ func (r RealRunner) Run(command string, args ...string) error {
 	fmt.Println(fmt.Sprintf("Provisioning OpenShift via '%s %s'", command, args))
 	cmd := exec.Command(command, args...)
 	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stdout
```

Comment:
So what will be the error message in the end? For the case the flag is not supported?